docs/servers/add-gcp-server.md

```diff
@@ -13,22 +13,23 @@ links:
 
 # Add a Server to Google Cloud Platform (GCP)
 1. On Devopness, navigate to a project then select an environment
-1. If there's no other server in the selected environment, just click Add Server
-1. If a list of resource cards is displayed, find the Servers card and click View on it
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
 1. On the upper-right corner of the list click `ADD SERVER`
 1. Choose `Google Cloud Platform` as cloud provider
 1. Select a GCP credential using the drop-down menu
-    > If you dont have any GCP credentials listed, follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide before next step
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide
```

Comment:
The same case of the link here. It's correct?

Reply:
fixed